lib/logging.js

```diff
@@ -98,7 +98,7 @@ export function displayErrorLog (errorLog) {
     console.log(`\n\nThe following ${errorsCount} errors and ${warningsCount} warnings occurred:\n`)
 
     errorLog
-      .map((logMessage) => `${moment(logMessage.ts).format('HH:mm:SS')} - ${formatLogMessageOneLine(logMessage)}`)
+      .map((logMessage) => `${format(logMessage.ts, 'HH:mm:ss')} - ${formatLogMessageOneLine(logMessage)}`)
```

Comment:
Couldn't find an exact match to test this - can confirm this is ok from unit test logs:

![image](https://user-images.githubusercontent.com/59754249/171586328-7fe635dc-2b85-44f2-a583-b85f5c515424.png)